.gitpod.yml

```diff
@@ -2,24 +2,14 @@ image:
   file: .gitpod.Dockerfile
 
 tasks:
-  - name: Futurenet
-    init: |
-      docker pull stellar/quickstart:soroban-dev
-    command: |
-      docker run --rm -it \
-      --name stellar \
-      -p 8000:8000 \
-      stellar/quickstart:soroban-dev \
-      --futurenet \
-      --enable-soroban-rpc
-  # This "CLI - Futurenet" task opens a terminal for you to interact with the
-  # futurenet network.
-  - name: CLI - Futurenet
-    # We specify some required environment variables for use on the futurenet.
+  # This "CLI - Testnet" task opens a terminal for you to interact with the
+  # testnet network.
+  - name: CLI - Testnet
```

Comment:
@kalepail What's your opinion on connection to testnet vs running a local quickstart. A local quickstart is faster to iterate on because it has 1 second ledgers, but deploy to testnet is probably also the most direct route to feeling the rush of success of deploying to a real network.

Reply:
Going to move ahead with this. Because I realise this wasn't running a local quickstart in isolation, but using futurenet, which I missed, and yeah folks should use testnet now.